NAMESPACE

```diff
@@ -13,4 +13,5 @@ export(isFalse)
 export(isTrue)
 export(not)
 export(test_hamcrest)
+export(throw_assert_error)
```

Comment:
Wait, you are now introducing a second code style to the project? Either change the rest of the functions, or stick to project's existing code style.

Reply:
Sure that.